#! /bin/sh /usr/share/dpatch/dpatch-run
## 06-revert-mpc-kokyu-change.dpatch by Thomas Girard <thomas.g.girard@free.fr>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use upstream patch to unbreak Kokyu library search path

@DPATCH@
--- ACE_wrappers.orig/bin/MakeProjectCreator/modules/AutomakeWorkspaceHelper.pm
+++ ACE_wrappers/bin/MakeProjectCreator/modules/AutomakeWorkspaceHelper.pm
@@ -135,7 +135,7 @@
         $reldir = 'Kokyu';
       }
     }
-    $str =~ s!$libname!\$(top_builddir)/$reldir/$libname!;
+    $str =~ s!$libname!\$(ACE_BUILDDIR)/$reldir/$libname!;
     return $str;
   }
   elsif ($libname =~ /^lib(tao|kokyu)/i) {
